search/script.js

```diff
@@ -0,0 +1,26 @@
+   var a = [0, 1, 2, 2, 3, 4, 11, 22, 69, 235, 459, 637, 789, 854];
+   a.sort(function(a, b) {
+     return a - b;
+   })
+
+   console.log(search(a, 235));
+   console.log(search(a, 0));
+   console.log(search(a, 854));
+
+   function search(array, value) {
+     return recursive(array, value, 0, array.length);
+   };
+
+   function recursive(array, value, left, right) {
```

Comment:
I'd suggest to move it under search function, it's helper and it shouldn't be in global scope